api/bazel/repository_locations.bzl

```diff
@@ -14,11 +14,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "protoc-gen-validate (PGV)",
         project_desc = "protoc plugin to generate polyglot message validators",
         project_url = "https://github.com/envoyproxy/protoc-gen-validate",
-        version = "872b28c457822ed9c2a5405da3c33f386ac0e86f",
-        sha256 = "388ea2261bc1d2c6ef6ec01bfaa3aec451aedb245e23514033ccc9b5cc10c4ab",
+        version = "6a6d869b7c0a587c56fba8322ae2724cc9115920",
+        sha256 = "c03881f9b6eb445eede5aa567ffa95ba7b0a0b4b840c93c921ee7447684334b4",
         strip_prefix = "protoc-gen-validate-{version}",
         urls = ["https://github.com/envoyproxy/protoc-gen-validate/archive/{version}.tar.gz"],
-        release_date = "2021-01-05",
+        release_date = "2021-03-09",
```

Comment:
This is not the latest commit. Can we move to the latest?
```
    com_envoyproxy_protoc_gen_validate = dict(
        project_name = "protoc-gen-validate (PGV)",
        project_desc = "protoc plugin to generate polyglot message validators",
        project_url = "https://github.com/envoyproxy/protoc-gen-validate",
        version = "b60bc972ba41dab246dbd0d45bb6a74d82e9ecd7",
        sha256 = "da5f32d925dd07b80a80769a06b0e0f632dff30a656fbd4ffacac2de4a306ffd",
        release_date = "2021-03-10",
        strip_prefix = "protoc-gen-validate-{version}",
        urls = ["https://github.com/envoyproxy/protoc-gen-validate/archive/{version}.tar.gz"],
        use_category = ["api"],
        implied_untracked_deps = [
            "com_github_iancoleman_strcase",
            "com_github_lyft_protoc_gen_star",
            "com_github_spf13_afero",
            "org_golang_google_genproto",
            "org_golang_x_text",
        ],
    ),
```

Reply:
Done.